UI/forms/OBSBasic.ui

```diff
@@ -425,7 +607,7 @@
   <widget class="OBSBasicStatusBar" name="statusbar"/>
   <widget class="OBSDock" name="scenesDock">
    <property name="features">
-    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetFloatable</set>
+    <set>QDockWidget::AllDockWidgetFeatures</set>
```

Comment:
Per [the Qt documentation](https://doc.qt.io/qt-5/qdockwidget.html), AllDockWidgetFeatures is deprecated in favour of manually definining all states. This was likely automatically swapped due to an older Qt Creator.

Reply:
Just for the record Qt Designer did that.

Reply:
I thought `QDockWidget::AllDockWidgetFeatures` was deprecated?  See #3050.

Reply:
Sorry, as usual, comments in email get confusing because they aren't marked outdated, and then the link brings me here.  Sorry for the noise.